.github/workflows/test.yml

```diff
@@ -51,7 +51,7 @@ jobs:
         with:
           python-version: ${{ matrix.python.version }}
       - name: Install dependencies
-        run: pip3 install tox pytest-github-actions-annotate-failures
+        run: pip3 install tox
```

Comment:
Maybe just `pip` (instead of `pip3`) like the rest of them?

Reply:
Done :)